Makefile

```
@@ -1,21 +1,43 @@
 .PHONY: help tests
 
+OS := $(shell uname -s)
+
 CONFIG_FILE?=config-files/config.yaml
 
+ifeq ($(OS),Linux)
+	JQ_INSTALL_CMD = sudo apt-get install jq
+	YQ_INSTALL_CMD = sudo apt-get install yq
```

there is no apt-get yq
```suggestion
	YQ_INSTALL_CMD = brew install yq
```